tt_metal/tt_metal.cpp

```diff
@@ -12,6 +12,7 @@
 #include <unordered_set>
 
 #include "dev_msgs.h"
+#include "hostdevcommon/common_runtime_address_map.h"
```

Comment:
Is there something in this file that uses this include? We're moving to get rid of this file

Reply:
removed